test/test_helper.rb

```diff
@@ -27,7 +27,8 @@
 # we can print one error and fail fast so that it's more clear to the user how
 # they should fix the problem.
 begin
-  resp = Faraday.get("http://localhost:#{MOCK_PORT}/")
+  conn = Faraday::Connection.new("http://localhost:#{MOCK_PORT}")
+  resp = conn.get("/")
```

Comment:
I had to change this because using the default connection "locked" the middleware stack and prevented me from changing the adapter above. I guess it only worked before by virtue of the fact that all the middlewares we installed were default and thus already installed.